specification/resources/resource-manager/Microsoft.Solutions/stable/2019-07-01/managedapplications.json

```diff
@@ -1316,11 +1324,70 @@
           "readOnly": true,
           "$ref": "#/definitions/ProvisioningState",
           "description": "The managed application provisioning state."
+        },
+        "billingDetails": {
+          "readOnly": true,
+          "$ref": "#/definitions/ApplicationBillingDetailsDefinition",
+          "description": "The managed application billing details."
+        },
+        "additionalManagedScopes": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The additional managed scopes outside of the normal managed resource group."
+        },
+        "JITAccessPolicy": {
```

Comment:
Mutable properties are considered a breaking change and should be in a new api-version.  See https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo

Reply:
We don't have published SDK yet, so the changes shouldn't be breaking anyone.

Reply:
Its a breaking API change.  2019-07-01 is live in your prod RP manifest

Reply:
Yeah, the manifest and our backend service code already support these properties in this version. 
The swagger is a bit out dated, and the previous PR created this swagger version never published or generated SDK.